common/static/js/vendor/pdfjs/locale/locale.properties

```diff
@@ -304,6 +304,9 @@
 [zh-CN]
 @import url(zh-CN/viewer.properties)
 
+[zh-HANS]
+@import url(zh-CN/viewer.properties)
```

Comment:
There are no distinct zh-hans translations, this is just mapping them to the existing zh-cn ones.